nixos/modules/security/systemd-confinement.nix

```diff
@@ -98,8 +98,9 @@ in {
         wantsAPIVFS = lib.mkDefault (config.confinement.mode == "full-apivfs");
       in lib.mkIf config.confinement.enable {
         serviceConfig = {
-          RootDirectory = "/var/empty";
-          TemporaryFileSystem = "/";
+          RuntimeDirectory = [ "confinement/${mkPathSafeName name}" ];
+          RootDirectory = lib.mkForce "/run/confinement/${mkPathSafeName name}";
```

Comment:
(discussed out-of-band)
I was under the impression that 'mkForce' is best left for users, so we'd probably want to use mkOverride 90 or some value between default value and mkForce for this, even if nobody should ever override this...

But it doesn't look like there's a well defined value for this, and I see plenty enough of mkForce in nixos/modules so this is probably fine? If a user *realy* wants to override this for some reason they can use an even lower priority.

For reference:
```
  mkOptionDefault = mkOverride 1500; # priority of option defaults
  mkDefault = mkOverride 1000; # used in config sections of non-user modules to set a default
  defaultOverridePriority = 100;
  mkImageMediaOverride = mkOverride 60; # image media profiles can be derived by inclusion into host config, hence needing to override host config, but do allow user to mkForce
  mkForce = mkOverride 50;
  mkVMOverride = mkOverride 10; # used by ‘nixos-rebuild build-vm’
```

Reply:
Why does this even need to use `mkForce` in the first place?